src/test/java/org/rulez/demokracia/PDEngine/VoteCreationTest.java

```diff
@@ -110,4 +111,340 @@ public void create_creates_a_vote_with_adminKey() {
 		assertEquals(VoteRegistry.getByKey(adminInfo.adminKey).adminKey, adminInfo.adminKey);
 	}
 
+// voteName validation tests
+
+	@Test
+	public void votename_contains_wrong_characters_input1() throws ReportedException {
```

Comment:
remember that test name will be part of the documentation.  be specific about what is tested
vote_name_should_not_contain_space